File: Code Python/Exercicio-2/Leitura de duas Notas.py
import os
n1 = float(input("Digite a sua primeira nota: \n"))
n2 = float(input("Digite a sua segunda nota: \n"))
media = (n1+n2)/2
if(media >= 7) and (media <= 9.9):
    print("Aprovado")
else:
    if(media < 7):
        print("Reprovado")
    else:
        if(media == 10):
            print("Aprovado com Distinção")
input("Pressione <enter> para sair")

File: Code_C/Exercicio-3/Leitura de duas Notas.cpp
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
int main(){
	
	float n1, n2, media;
	
	printf("Digite a sua primeira nota: \n");
	scanf("%f", &n1);
    printf("Digite a sua segunda nota: \n");
    scanf("%f", &n2);
    
media = (n1+n2)/2;
	
	if((media >= 7)&&(media <= 9.9)){
	      printf("Aprovado \n");
	}else{
	    if(media < 7){
	      printf("Reprovado \n");
	    }else{
	        if(media == 10){
	            printf("Aprovado com Distincao \n");
            }
	    }   
    }
system("PAUSE");
return(0);
}
